service/matching/taskQueueManager.go

```diff
@@ -282,7 +292,7 @@ func (c *taskQueueManagerImpl) AddTask(
 ) (bool, error) {
 	if params.forwardedFrom == "" {
 		// request sent by history service
-		c.liveness.markAlive(time.Now())
+		c.lastActivity.Store(c.clock.Now())
```

Comment:
wrap this is a function?

Reply:
I thought about it, but ended up not being able to describe what abstraction the function would provide. How it would justify its own existence. Is the intent here unclear from your perspective?

Reply:
I would suggest the same to wrap this in a func. Not that this is hard to understand, but I do have to look up what the lastActivity is, and realize the intention is to mark liveness. It would be nicer to just call:
c.markAlive() and it internally calls c.lastActivity.Store(c.clock.Now()).